legacy/ui/legacy/src/main/res/values/strings.xml

```diff
@@ -648,7 +648,8 @@
     <string name="animations_title">Animation</string>
     <string name="animations_summary">Use gaudy visual effects</string>
 
-    <string name="volume_navigation_title">Volume key navigation in message view</string>
+    <string name="volume_navigation_title_short">Volume key navigation</string>
+    <string name="volume_navigation_summary">Scroll through the message list using the volume up/down keys</string>
```

Comment:
Volume key navigation only works in message view to display the next/previous message.